noxfile.py

```diff
@@ -1,45 +1,36 @@
+#!/usr/bin/env uv run
```

Comment:
I think maybe `-S` is required on some OSs?

https://simonwillison.net/2024/Aug/21/usrbinenv-uv-run/

```suggestion
#!/usr/bin/env -S uv run
```